tools/mksurfdata_esmf/src/mksurfdata.F90

```diff
@@ -1242,15 +1307,6 @@ subroutine normalize_and_check_landuse(ns_o)
 
       end do
 
-      ! Make sure that there is no vegetation outside the pft mask
```

Comment:
Why was this removed?

Reply:
Good question. This error check felt like it wasn't serving much purpose anymore, because the new code in the above block - starting at line 1168 - explicitly zeroes out the vegetation in this case. If I remember correctly, this might have served more of a purpose before my changes, because the way this was done might have been less direct at that point.

Reply:
OK